### PR TITLE
Fail on CJS

### PR DESCRIPTION
**Pull request by sigma-andex**
- Emit error if a cjs ffi is detected
- Emit warning if parsing fails

**Description of the change**

Clearly and concisely describe the purpose of the pull request. If this PR relates to an existing issue or change proposal, please link to it. Include any other background context that would help reviewers understand the motivation for this PR.

---

**Checklist:**

- [ ] Added a file to CHANGELOG.d for this PR (see CHANGELOG.d/README.md)
- [ ] Added myself to CONTRIBUTORS.md (if this is my first contribution)
- [ ] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

File: src/Language/PureScript/Errors.hs [hunk 1]
```diff
@@ -713,7 +713,7 @@ prettyPrintSingleError (PPEOptions codeColor full level showDocs relPath) e = fl
     renderSimpleErrorMessage (DeprecatedFFICommonJSModule mn path) =
       paras [ line $ "A CommonJS foreign module implementation was provided for module " <> markCode (runModuleName mn) <> ": "
             , indent . lineS $ path
-            , line "CommonJS foreign modules are deprecated and won't be supported in the future."
+            , line "CommonJS foreign modules are deprecated and not supported anymore."
```
Comment (thomashoneyman) [inline src/Language/PureScript/Errors.hs]:
```suggestion
            , line "CommonJS foreign modules are no longer supported."
```

**Comment by sigma-andex** [comment]
Some bundle tests are now failing because cjs is detected. What do we do about them?

**Comment by thomashoneyman** [comment]
I think failing bundle tests can be removed and we can comment on the PR about the removal.

**Comment by JordanMartinez** [comment]
One reason I haven't taken any action on this is because I wasn't sure if this PR was intended to be merged into the base branch immediately, or if it was work being done so that, after the base branch gets merged into PureScript's `master` branch, this one is not too far behind.

**Comment by sigma-andex** [comment]
@JordanMartinez 
so you think it is better having this as a separate, later PR?

**Comment by JordanMartinez** [comment]
That's not quite what I mean. It's more, I don't know what the full changes we want to make are and what order in which to make them. For example, are we supporting ES modules initially in one PR but not dropping CommonJS modules yet in that PR? And then later dropping CommonJS modules in a new PR? And then dropping `purs bundle` in a final PR? Or is that all being mushed together?
My comment is illustrating my unfamiliarity with what we need to do.

**Comment by sigma-andex** [comment]
I guess usually it makes more sense to have multiple smaller PRs. So I guess that's perfectly fine and we could retarget this PR to purescript/master once the es-modules PR gets merged.

**Comment by sigma-andex** [comment]
Closing this to make way for a new small PR against `purescript/master`

**Comment by JordanMartinez** [comment]
Looks like this PR can be submitted to `master` directly now. As agreed upon in the call a few days ago, we're not going to warn if `language-javascript` parsing fails.

**Pull request closed without merging**